fv3_cap.F90

```diff
@@ -140,7 +140,7 @@ subroutine SetServices(gcomp, rc)
 
     ! specializations required to support 'inline' run sequences
     call NUOPC_CompSpecialize(gcomp, specLabel=label_CheckImport, &
-                              specPhaseLabel="phase1", specRoutine=NUOPC_NoOp, rc=rc)
+                              specPhaseLabel="phase1", specRoutine=fv3_checkimport, rc=rc)
```

Comment:
So now the fv3_checkImport will be called once for one integration step with two phases, just as it is called once in "ModelAdvance". One question is: should we also call checkImport for phase2? or maybe it is no necessary as the time stamp does not change?

Reply:
`fv3_checkimport` should be called at the beginning of the atmosphere's integration step for coupling with external components except GOCART. It is not called before run phase 2 since: (a) the timestamp of imported fields does not change, and (b) GOCART does not require it.